cg/apps/demultiplex/sample_sheet/sample_sheet_validator.py

```diff
@@ -156,46 +156,63 @@ def _validate_override_cycles(self) -> None:
             except OverrideCyclesError as error:
                 raise SampleSheetError from error
 
+    def _validate_sample_sheet_is_correct_type(self, bcl_converter: str) -> None:
+        """Determine if the sample sheet content corresponds to the provided converter.
+        Raises:
+            SampleSheetError if the sample sheet is not of the correct type.
+        """
+        expected_type: Type[FlowCellSample] = BCL_CONVERTER_TO_FLOW_CELL_SAMPLE[bcl_converter]
```

Comment:
Should the expected type not always be `BCL_CONVERTER`?

Reply:
Not yet @seallard. I am leaving manual creation and validation of BCL2FASTQ sample sheets still available for a while just in case. This PR only removes BCL2FASTQ from the automation in production